backtrace.h

```diff
@@ -7,13 +7,13 @@ modification, are permitted provided that the following conditions are
 met:
 
     (1) Redistributions of source code must retain the above copyright
-    notice, this list of conditions and the following disclaimer. 
```

Comment:
OK; removed this bit and will send a separate PR with the updates from upstream.